http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -488,6 +501,10 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         raw"relaylistener.runtimeName=${appName.value}",
         raw"relaylistener.image=${config.listenerImage}",
         raw"""relaylistener.removeEntityPathFromHttpUrl="${removeEntityPathFromHttpUrl.toString}"""",
+        raw"relaylistener.validHosts[0]=${validHosts(0)}",
+        raw"relaylistener.validHosts[1]=${validHosts(1)}",
+        raw"relaylistener.validHosts[2]=${validHosts(2)}",
+        raw"relaylistener.validHosts[3]=${validHosts(3)}",
```

Comment:
Should this be usable with an unknown number of valid hosts?

Reply:
does the valid hosts remain the same regardless of environment? If not, yeah I'll have to change this